Makefile

```diff
@@ -1,9 +1,9 @@
-HOSTNAME=kentik.com
-NAMESPACE=automation
-NAME=kentik-cloudexport
-BINARY=terraform-provider-${NAME}
-VERSION=0.1.0
-OS_ARCH=linux_amd64
+HOSTNAME = kentik
```

Comment:
Please adjust source and version of provider in `provider.tf` file in examples.

```terraform
terraform {
  required_providers {
    kentik-cloudexport = {
      version = "~> 0.2"
      source  = "kentik/automation/kentik-cloudexport"
    }
  }
}
```

`~> 0.2` will pick up 0.2 or newer.

Reply:
Good point. I'll do that in the next commit. I'll also create pull-request for "config snippets" to bump the TF provider version.